File: src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiDisconnect.java
package me.zeroeightsix.kami.module.modules.misc;

import me.zeroeightsix.kami.module.Module;
import me.zeroeightsix.kami.setting.Setting;
import me.zeroeightsix.kami.setting.Settings;

@Module.Info(
          name = "AntiDisconnect",
          description = "Are you sure you want to disconnect?",
          category = Module.Category.MISC
)
public class AntiDisconnect extends Module {
    public Setting<Integer> requiredButtonPresses = register(Settings.integerBuilder("Button Presses").withMinimum(1).withMaximum(20).withValue(6));
}

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiDisconnect.kt
package me.zeroeightsix.kami.module.modules.misc

import me.zeroeightsix.kami.module.Category
import me.zeroeightsix.kami.module.Module

internal object AntiDisconnect : Module(
    name = "AntiDisconnect",
    description = "Are you sure you want to disconnect?",
    category = Category.MISC
) {
    val presses = setting("ButtonPresses", 3, 1..20, 1)
}